pkg/handlers/replicas.go

```diff
@@ -48,10 +48,6 @@ func MakeReplicaUpdater(defaultNamespace string, clientset *kubernetes.Clientset
 			}
 		}
 
-		if len(req.ServiceNamespace) > 0 {
```

Comment:
This looks unrelated to the renames. Was this change expected in this pr?

Reply:
It is related in so far as not needing to temporarily create a `pkg/types/requests.go`

Reply:
This is in the commit message:

```
This patch also removes the service namespace field from
the scale type. The value can and is read from the query
string, meaning that this field was redundant and non-standard
compared with the types in faas-provider.
```

Reply:
Not an issue on this occasion, but thanks for the keen eye, this is why reviews are useful 👍